README.md

```diff
@@ -207,7 +207,7 @@ $index->search(
   ],
   "offset": 0,
   "limit": 20,
-  "nbHits": 1,
```

Comment:
Just to remind myself, but we need to deal with the `nbHits` in the meilisearch-php to prevent the breaking in the laravel/scout!